dev-packages/node-integration-tests/suites/tracing/redis-cache/scenario-redis-4.js

```diff
@@ -0,0 +1,54 @@
+const { loggingTransport } = require('@sentry-internal/node-integration-tests');
+const Sentry = require('@sentry/node');
+
+Sentry.init({
+  dsn: 'https://public@dsn.ingest.sentry.io/1337',
+  release: '1.0',
+  tracesSampleRate: 1.0,
+  transport: loggingTransport,
+  integrations: [Sentry.redisIntegration({ cachePrefixes: ['redis-cache:'] })],
+});
+
+// Stop the process from exiting before the transaction is sent
+setInterval(() => {}, 1000);
+
+const { createClient } = require('redis-4');
+
+async function initializeClient() {
+  return createClient().connect();
+}
+
+let client;
```

Comment:
l: Can we rename this (and the method below) to e.g. `redisClient`? Can be a bit confusing otherwise as we also have a sentry client :)